cluster/charts/rook-ceph/templates/serviceaccount.yaml

```diff
@@ -8,71 +13,83 @@ metadata:
     operator: rook
     storage-backend: ceph
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
-{{ template "imagePullSecrets" . }}
+{{ template "imagePullSecrets" $ }}
```

Comment:
What is the difference between `$` and `.` with the template?

Reply:
The change is for pointing root context.

As shown in [the official document](https://v2.helm.sh/docs/chart_template_guide/#variables), inside of looping, the current context(i.e. "." ) is overwritten and .Values.{something} can not be used.
> However, there is one variable that is always global - $ - this variable will always point to the root context. This can be very useful when you are looping in a range need to know the chart's release name.

Therefore, inside of the loop, I used the root context, "$", because, as it is root,  $.Values.{something} is always valid.